code/bmpman/bmpman.cpp

```diff
@@ -3151,6 +3243,8 @@ void bm_unload_all() {
 	}
 
 	Bm_ignore_load_count = 0;
+
+	vm_free(Pathtypes[CF_TYPE_TEMP_SUBDIR_LOOKUP].path);
```

Comment:
Shouldn't this be in `cfile_close()` since the initial `vm_strdup()` call is in `cfile_init()`?